components/question/Details/index.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+import useUser from 'hooks/useUser';
+import useSurvey from 'hooks/useSurvey';
+
+const QuestionDetails = ({ setBackgroundImagePath }) => {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
```suggestion
  const questionId = router.query.id;
```

Similar to a previous comment ;-)

Reply:
done in 0a12e6f2768d6cf6c93db4a407b806b7fc73a764